package-lock.json

```diff
@@ -19,9 +19,9 @@
       "integrity": "sha512-UFMC4ZeFC48Tpvj7C8UgLvtkaUuovQX+5xNWrsIoMG8o2z+XFKjKaN9iVmS84dPwVN00W4wPmqvYoZF3EGAsfw==",
       "dev": true,
       "requires": {
-        "chalk": "^2.0.0",
-        "esutils": "^2.0.2",
-        "js-tokens": "^4.0.0"
+        "chalk": "2.4.2",
```

Comment:
Why the specificity?  I get the idea of wanting a consistent build, but this makes it harder to perform routine updates.

Reply:
Not sure why this is like this.   I just did `yarn install` after cloning the repo